### PR TITLE
added resize

### PR DESCRIPTION
**Pull request by blublinsky**

### CODE REVIEW

File: transforms/universal/resize/python/Makefile
Comment (daw3rd) [inline transforms/universal/resize/python/Makefile:1]: I would reocmmend restarting this file and based it on noop's python Makefile.  It is quite different from this.

File: transforms/universal/resize/ray/Makefile
Comment (daw3rd) [inline transforms/universal/resize/ray/Makefile:1]: Also, would recommend restarting this file from. noop ray/Makefile.

File: transforms/universal/resize/ray/Readme.md
Comment (daw3rd) [inline transforms/universal/resize/ray/Readme.md:1]: Can we rename this to README.md for consistency.

File: transforms/universal/resize/ray/test/test_resize_launch.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    """
+    Extends the super-class to define the test data for the tests defined there.
+    The name of this class MUST begin with the word Test so that pytest recognizes it as a test class.
+    """
+
+    def get_test_transform_fixtures(self) -> list[tuple]:
+        # The following based on 3 identical input files of about 39kbytes, and 200 rows
+        common_config = {"runtime_num_workers": 1, "run_locally": True}  # to make the output files repeatable.
+        fixtures = []
+        basedir = os.path.abspath(os.path.join(os.path.dirname(__file__), "../test-data"))
```
Comment (daw3rd) [inline transforms/universal/resize/ray/test/test_resize_launch.py:32]: would it be worth trying ../../python/test-data here?

**Review by daw3rd — CHANGES_REQUESTED** [review]
Also needs a python/Dockerfile

**Review by daw3rd — CHANGES_REQUESTED** [review]
You need a resize/Makefile and REAMDE.md.  Until this Makefile is added, no testing of resize is happening :).

File: transforms/universal/resize/ray/test/test_resize_launch.py (hunk 1 above)
Comment (blublinsky) [reply transforms/universal/resize/ray/test/test_resize_launch.py:32]: yes. it will cause issues in docker

File: transforms/universal/resize/ray/Readme.md
Comment (blublinsky) [reply transforms/universal/resize/ray/Readme.md:1]: Done

File: transforms/universal/resize/README.md
Comment (daw3rd) [inline transforms/universal/resize/README.md:1]: This is the wrong readme.  You need the one that lists the runtimes.  See any other transform.

File: transforms/universal/resize/ray/Makefile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+publish:: publish-dist publish-image
+
+publish-image:: .transforms.publish-image-ray
+
+setup:: .transforms.setup
+
+# set the version of python transform that this depends on.
+set-versions:
+	$(MAKE) TRANSFORM_PYTHON_VERSION=${DOCKER_IMAGE_VERSION} TOML_VERSION=$(DOCKER_IMAGE_VERSION) .transforms.set-versions
```
Comment (daw3rd) [inline transforms/universal/resize/ray/Makefile]: you can remove the TOML_VERSION setting

File: transforms/universal/resize/python/Makefile [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+publish:: publish-dist publish-image
+
+publish-image:: .transforms.publish-image-python
+
+setup:: .transforms.setup
+
+# distribution versions is the same as image version.
+set-versions:
+	$(MAKE) TRANSFORM_PYTHON_VERSION=${DOCKER_IMAGE_VERSION} TOML_VERSION=$(DOCKER_IMAGE_VERSION) .transforms.set-versions
```
Comment (daw3rd) [inline transforms/universal/resize/python/Makefile]: You can remove the TOML_VERSION setting.

File: transforms/universal/resize/ray/src/resize_transform_ray.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    def __init__(self):
+        """
+        Initialization
+        """
+        super().__init__(transform_config=ResizeTransformConfiguration())
+
+
+if __name__ == "__main__":
+    # launcher = NOOPRayLauncher()
```
Comment (daw3rd) [inline transforms/universal/resize/ray/src/resize_transform_ray.py]: maybe get rid of this comment?

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: transforms/universal/resize/README.md
Comment (blublinsky) [reply transforms/universal/resize/README.md:1]: Fixed

File: transforms/universal/resize/ray/Makefile (hunk 2 above)
Comment (blublinsky) [reply transforms/universal/resize/ray/Makefile]: done

File: transforms/universal/resize/python/Makefile (hunk 3 above)
Comment (blublinsky) [reply transforms/universal/resize/python/Makefile]: done

File: transforms/universal/resize/ray/src/resize_transform_ray.py (hunk 4 above)
Comment (blublinsky) [reply transforms/universal/resize/ray/src/resize_transform_ray.py]: done

**Review by daw3rd — APPROVED** [review]

**Pull request merged**